### PR TITLE
Issue7 - Enrich line item payload

### LINKED ISSUE #7: Checkout Payload anpassen?
Payload erweitern um die E-Mail Adresse, Daten/Datum, ggf. templateId

### PR DESCRIPTION
**Pull request by seggewiss**
Resolves #7

### CODE REVIEW

File: src/Resources/views/storefront/page/product-detail/buy-widget-form.html.twig [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+               name="{{ constant('Swag\\ServiceBooking\\Storefront\\Controller\\ServiceBookingCartController::SERVICE_BOOKING_TEMPLATE_REQUEST_PARAMETER') }}[id]"
+               value="{{ bookingTemplate.id }}">
+
+        <input type="hidden"
+               name="{{ constant('Swag\\ServiceBooking\\Storefront\\Controller\\ServiceBookingCartController::SERVICE_BOOKING_TEMPLATE_REQUEST_PARAMETER') }}[dateId]"
+               value="">
+    {% endif %}
+{% endblock %}
+
+{% block page_product_detail_buy_form_action %}{% apply spaceless %}
```
Comment (King-of-Babylon) [inline src/Resources/views/storefront/page/product-detail/buy-widget-form.html.twig:19]: Einrückung?

File: src/Resources/views/storefront/page/product-detail/buy-widget-form.html.twig [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    {% if bookingTemplate is not null %}
+        {{ path('frontend.checkout.service-booking.add') }}
+    {% else %}
+        {{ parent() }}
+    {% endif %}
+{% endapply %}
```
Comment (King-of-Babylon) [inline src/Resources/views/storefront/page/product-detail/buy-widget-form.html.twig:25]: Hier auch?

**Review by King-of-Babylon — APPROVED** [review]

File: src/Storefront/Controller/ServiceBookingCartController.php [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        $this->cartService = $cartService;
+    }
+
+    /**
+     * @Route("/checkout/service-booking/add", name="frontend.checkout.service-booking.add", methods={"POST"}, defaults={"XmlHttpRequest"=true})
+     *
+     * @throws MissingRequestParameterException
+     * @throws InvalidUuidException
+     */
+    public function addCustomizedProduct(
```
Comment (SebastianFranze) [inline src/Storefront/Controller/ServiceBookingCartController.php]: addCustomizedProduct? :P

Comment (seggewiss) [reply src/Storefront/Controller/ServiceBookingCartController.php]: Whoopsie :D Done <3

**Review by SebastianFranze — APPROVED** [review]

**Pull request merged**